Packs/SophosXGFirewall/Integrations/sophos_firewall.yml

```
@@ -0,0 +1,4166 @@
+    - auto: PREDEFINED
+      description: Enable policy rule
+      isArray: false
+      name: policy_rule_enabled
+      predefined:
+      - '1'
+      - '0'
+      required: false
+      secret: false
+    - description: Choose users which this rule will apply on
+      isArray: false
+      name: user_names
+      required: false
+      secret: false
+    - description: 'CCL names. REQUIRED: when ccl_rule_enabled is ON'
+      isArray: false
+      name: ccl_names
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      description: 'Enable CCL rule. IMPORTANT: if enabled - ccl_name is requierd.'
+      isArray: false
+      name: ccl_rule_enabled
+      predefined:
+      - '1'
+      - '0'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      description: Enable following HTTP action
+      isArray: false
+      name: follow_http_action
+      predefined:
+      - '1'
+      - '0'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: true
+      defaultValue: Enable
+      description: Select to enable reporting of policy.
+      isArray: false
+      name: enable_reporting
+      predefined:
+      - Enable
+      - Disable
+      required: false
+      secret: false
+    deprecated: false
+    description: Update an existing web filter policy.
+    execution: false
+    name: sophos-firewall-web-filter-update
+    outputs:
+    - contextPath: SophosFirewall.WebFilterPolicy.Name
+      description: Name of the policy.
+      type: String
+    - contextPath: SophosFirewall.WebFilterPolicy.DefaultAction
+      description: Default action for the web filter policy.
+      type: String
+    - contextPath: SophosFirewall.WebFilterPolicy.Description
+      description: Description of the rule.
+      type: String
+    - contextPath: SophosFirewall.WebFilterPolicy.EnableReporting
+      description: Does the policy report events.
+      type: String
+    - contextPath: SophosFirewall.WebFilterPolicy.DownloadFileSizeRestriction
+      description: Maximum file size that can be downloaded.
+      type: Number
+    - contextPath: SophosFirewall.WebFilterPolicy.DownloadFileSizeRestrictionEnabled
+      description: Is the file size restriction active.
+      type: String
+    - contextPath: SophosFirewall.WebFilterPolicy.RuleList.Rule
+      description: Rule list information.
+      type: String
+  - arguments:
+    - default: false
+      description: Name of the policy.
+      isArray: false
+      name: name
+      required: true
+      secret: false
+    deprecated: false
+    description: Delete an existing web filter policy.
+    execution: false
+    name: sophos-firewall-web-filter-delete
+    outputs:
+    - contextPath: SophosFirewall.WebFilterPolicy.Name
+      description: Name of the policy.
+      type: String
+    - contextPath: SophosFirewall.WebFilterPolicy.IsDeleted
+      description: Whether or not the policy was deleted.
+      type: String
```

Please change for all the `IsDeleted` keys
```suggestion
      type: Bool
```